server/repository/src/db_diesel/clinician_link_row.rs

```diff
@@ -115,6 +115,16 @@ impl<'a> ClinicianLinkRowRepository<'a> {
         Ok(result)
     }
 
+    pub fn find_many_by_clinician_id(
+        &self,
+        clinician_id: &str,
+    ) -> Result<Vec<ClinicianLinkRow>, RepositoryError> {
+        let result = clinician_link_dsl::clinician_link
+            .filter(clinician_link::clinician_id.eq(clinician_id))
+            .load::<ClinicianLinkRow>(&self.connection.connection)?;
+        Ok(result)
```

Comment:
Is there a reason not to use the `ClinicianRepository::query_by_filter` method? if not could you please remove this one again and use it?

Reply:
Because this is loading the `ClinicianLinkRow`?

Reply:
Yes please ignore